src/main/resources/repos.csv

```diff
@@ -0,0 +1,21 @@
+cloneUrl,branch,org1,orgN
```

Comment:
I thought `orgN` was an example, not real. I was expecting `org1,org2,org3,org4...`, not to have an actual `orgN`.

Reply:
I was not sure how to put this in the header, but we can use basically anything as we skip it anyway.

Reply:
updated